tests/integration/components/nav-bar-test.js

```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | nav-bar', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
+
+    await render(hbs`<NavBar />`);
+
+    assert.equal(this.element.textContent.trim(), '');
```

Comment:
See you're rendering Navbar, it has text `Home` and `Signup` now
And you're checking if that's equal to `''` which is causing the tests to fail and hence the github checks to fail
Please check